### ISSUE TITLE
az acr helm push --force won't push if it doesn't exists

### ISSUE THREAD

**Issue by ferantivero** [Azure/azure-cli#7778]
**Is your feature request related to a problem? Please describe.**
to some extent... it could be for CI contexts.

**Describe the solution you'd like**
I was expecting that --force could override or create a chart if it doesn't exists: 
```az acr helm push <NEW_CHART_NAME> --force -n <acr name>```

The problem is that if the package does not exists you get the following. It is aligned with the docs but not practical:

>msrest.http_logger : Response content:
>msrest.http_logger : b'{"errors":[{"code":"CHART_NOT_FOUND","message":"You have requested chart that does not exist."}]}\n' 

**Describe alternatives you've considered**
currently I'm doing a query first then push with/without force. I think it is better if force could just push when it doesn't exists.

**Additional context**
In my opinion, it could be beneficial for CI pipelines.

**Comment by mathieu-benoit** [comment]
FYI: https://github.com/Azure/acr/issues/142

**Comment by SteveLasker** [comment]
Agreed. We’re tracking this in the linked repo. Suggest closing this issue
